### PR TITLE
WEB-261: Add small docs updates and small change for files.

### PR DESCRIPTION
**Pull request by StavoveiC**

### CODE REVIEW

File: files_daily.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ if [[ ! -v PROJECT_NAME ]]; then
 fi
 
 # check how many sites (ignore [^ name created by using pattern in mounts)
-NUMBER_OF_SITES=$(find $SITES_DIR -maxdepth 0 -type d ! -name '[^' | wc -l)
+NUMBER_OF_SITES=$(find $SITES_DIR -maxdepth 0 -type d | wc -l)
```
Comment (ivangrozni) [inline files_daily.sh:20]: use double quote to prevent globbing (also on next few lines)

File: platform_db_backup.sh [hunk 2]
```diff
@@ -61,23 +61,23 @@ if [[ -v AWS_BACKUP_BUCKET || -v SFTP_SERVER ]]; then
     if [[ -v SFTP_DIRECTORY && -v SITE && -v SFTP_SERVER ]]; then
       echo "Checking old backups"
       if [[ -n "$SFTP_PORT" ]]; then
-        ssh -p $SFTP_PORT ${SFTP_USERNAME}@${SFTP_SERVER} "find ~/$SFTP_DIRECTORY/drush-backups/$PROJECT_NAME -mindepth 1 -type d -mtime +${SFTP_DAYS_EXP:180} -printf '%p\n' |grep -v '\-d01' |xargs -I {} rm -r -v \"{}\""
+        ssh -p "$SFTP_PORT" "${SFTP_USERNAME}"@"${SFTP_SERVER}" "find ~/$SFTP_DIRECTORY/drush-backups/$PROJECT_NAME -mindepth 1 -type d -mtime +${SFTP_DAYS_EXP:180} -printf '%p\n' |grep -v '\-d01' |xargs -I {} rm -r -v \"{}\""
```
Comment (ivangrozni) [inline platform_db_backup.sh:64]: SC1117: Backslash is literal in "\n". Prefer explicit escaping: "\\n". (`printf '%p\n'`)

File: platform_db_backup.sh [hunk 3]
```diff
@@ -61,23 +61,23 @@ if [[ -v AWS_BACKUP_BUCKET || -v SFTP_SERVER ]]; then
...
       else
-        ssh ${SFTP_USERNAME}@${SFTP_SERVER} "find ~/$SFTP_DIRECTORY/drush-backups/$PROJECT_NAME -mindepth 1 -type d -mtime +${SFTP_DAYS_EXP:180} -printf '%p\n' |grep -v '\-d01' |xargs -I {} rm -r -v \"{}\""
+        ssh "${SFTP_USERNAME}"@"${SFTP_SERVER}" "find ~/$SFTP_DIRECTORY/drush-backups/$PROJECT_NAME -mindepth 1 -type d -mtime +${SFTP_DAYS_EXP:180} -printf '%p\n' |grep -v '\-d01' |xargs -I {} rm -r -v \"{}\""
```
Comment (ivangrozni) [inline platform_db_backup.sh:66]: same as above

**Review by ivangrozni — APPROVED** [review]
@StavoveiC looks good to me. I still see some cosmetic comments if I run the code into shellcheck but nothing that stands out.

File: files_daily.sh (hunk 1 above)
Comment (StavoveiC) [reply files_daily.sh:20]: doesn't work here. Tried it already. if I used "" it won't find the directory

File: platform_db_backup.sh (hunk 3 above)
Comment (StavoveiC) [reply platform_db_backup.sh:66]: doesn't work when the expression is already with ""

**Pull request merged**